tensorrt_llm/_torch/model_config.py

```
@@ -527,3 +529,17 @@ def get_num_attention_layers(self):
+
+    def clone(self) -> "ModelConfig[TConfig]":
+        """
+        Create a clone of the config.
+        """
+        shallow_fields = ["mapping"]
+
+        clone = dataclasses.replace(self,
+                                    **{field: None
+                                       for field in shallow_fields})
+        clone = copy.deepcopy(clone)
+        for field in shallow_fields:
```

Ruff (0.12.2)

Import `field` from line 5 shadowed by loop variable

(F402)